doc/sphinx/administration_portal/brand/settings/client_portals.rst

```diff
@@ -1,5 +1,7 @@
+.. _client_portals:
+
 ############
-Portals URLs
+Client Portals
 ############
```

Comment:
Title `######` must be long to fill all title text. This applies to other titles too, otherwise sphinx will output warning.

Reply:
Fixed. Thanks Kai